pom.xml

```diff
@@ -16,7 +16,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <grpc.version>1.42.1</grpc.version>
     <protobuf.version>3.17.3</protobuf.version>
-    <dapr.proto.baseurl>https://raw.githubusercontent.com/dapr/dapr/683a436ff1367e4ff2d27da2d79069c04ec2c46d/dapr/proto</dapr.proto.baseurl>
+    <dapr.proto.baseurl>https://raw.githubusercontent.com/dapr/dapr/master/dapr/proto</dapr.proto.baseurl>
```

Comment:
should we always only point to master? In releases this is always set to a particular commit hash for reproducibility of build.... can we continue that for normal master builds as well ?

Reply:
Same here. We don't point to master because we want the build to be predictable.

Reply:
Updated back to ref `68a43...` in the pom for now; regarding reproducibility, should we consider doing something similar to the k8s Java client (and others) and simply add the generated source files to git? It would avoid downloading temporary protobuf files and also make it clear which source files are being used / generated for a build if there is a problem.